app/helpers/validate/throwMissingParameter.spec.js

```diff
@@ -0,0 +1,48 @@
+const assert = require("assert")
+const throwMissingParameter = require("./throwMissingParameter");
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/ufabc-next/ufabc-next-server/pullRequest?prid=8555381)
```suggestion
const throwMissingParameter = require('./throwMissingParameter');
```